hooks/src/lib/useCVCNumber/useCVCNumber.ts

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+import { CVC_NUMBER_ERROR_TYPE } from './useCVCNumber.constant';
+import useCVCNumberValidation from './useCVCNumberValidation';
+
+const useCVCNumber = () => {
+  const [cvcNumber, setCVCNumber] = useState('');
```

Comment:
DX를 고려하여 개발자가 `initialState`를 직접 입력받게 해도 좋을 것 같아요!

```suggestion
const useCVCNumber = (initialState = '') => {
  const [cvcNumber, setCVCNumber] = useState(initialState);
```